### PR TITLE
Sasquatch ui matches Puppet ui.

### PR DESCRIPTION
**Pull request by bmourat**
Sasquatch app now has same ui as Puppet app.

### CODE REVIEW

**Comment by TroubleMakerBen** [comment]
Looking good to me!

File: Sasquatch/Sasquatch/MobileCenterProtocol.swift [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
+ * Protocol that all ViewControllers interacting with MobileCenter should implement.
+ */
+
```
Comment (jaelim-ms) [inline Sasquatch/Sasquatch/MobileCenterProtocol.swift]: Remove empty line.

File: Sasquatch/Sasquatch/MobileCenterProtocol.swift [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*
```
Comment (jaelim-ms) [inline Sasquatch/Sasquatch/MobileCenterProtocol.swift]: Should be `/**`

File: Sasquatch/Sasquatch/Sasquatch-Bridging-Header.h [hunk 3]
```diff
@@ -3,3 +3,4 @@
 //
 
 #import "MSAlertController.h"
+#import "CrashLib.h"
```
Comment (jaelim-ms) [inline Sasquatch/Sasquatch/Sasquatch-Bridging-Header.h]: Reorder imports

**Comment by bmourat** [comment]
Don't have real device now, i'll fix it tomorrow!

**Comment by bmourat** [comment]
Crash on the real device fixed.

**Comment by jaelim-ms** [comment]
The apps don't send any logs.
I can see `Could not successfully update network info during initialization.` in the log, looks the apps have an issue to get network state.

**Comment by MatkovIvan** [comment]
@jaelim-ms do you use some network spy or just logging?
Looks like just verbose logging is disabled. I have logs sent.
Could you please provide some more info for reproduce this problem?

**Comment by jaelim-ms** [comment]
My bad @bmourat. Due to the default log level setting it didn't print out logs.

**Comment by MatkovIvan** [comment]
Guys, I have some improvements to this PR (https://github.com/bmourat/mobile-center-sdk-ios/pull/2), if you OK to merge now I may create separate PR for this.

**Comment by jaelim-ms** [comment]
Can we do that in a separate PR?

**Pull request merged**